Multisig/UI/App/Onboarding/OnboardingStepCollectionViewCell.swift

```diff
@@ -17,8 +17,16 @@ class OnboardingStepCollectionViewCell: UICollectionViewCell {
     static let identifier = "OnboardingStepCollectionViewCell"
 
     func configure(step: OnboardingStep) {
-        titleLabel.text = step.title
-        titleLabel.setStyle(.title1)
+        if let highlightedText = step.title.highlightedText {
+            titleLabel.attributedText = step.title.text.highlightRange(
+                originalStyle: .title1,
+                highlightStyle: .title1.color(.primary),
+                textToHighlight: highlightedText
+            )
+        } else {
+            titleLabel.text = step.title.text
```

Comment:
Is there a situation when there's no highlighted text provided?

Reply:
General case yes, but actually we don't have it right now